libraries/octree/src/OctreeQueryNode.cpp

```diff
@@ -182,14 +182,15 @@ bool OctreeQueryNode::updateCurrentViewFrustum() {
                                                              getCameraAspectRatio(),
                                                              getCameraNearClip(),
                                                              getCameraFarClip()));
+            newestViewFrustum.calculate();
         }
         
         
         { // if there has been a change, then recalculate
             QMutexLocker viewLocker(&_viewMutex);
             if (!newestViewFrustum.isVerySimilar(_currentViewFrustum)) {
                 _currentViewFrustum = newestViewFrustum;
-                _currentViewFrustum.calculate();
+                //_currentViewFrustum.calculateProjection();
```

Comment:
Oops.  I'll remove this crufty line.  We don't need to `calculate()` now that `newestViewFrustum` is no longer sometimes corrupt.